tensorflow/contrib/lite/examples/ios/simple/ios_image_load.h

```diff
@@ -16,10 +16,8 @@
 #define TENSORFLOW_EXAMPLES_IOS_IOS_IMAGE_LOAD_H_
```

Comment:
I would remove the header guards since they are non-standard ObjC and this header file is now no longer a C++ header.